source/ranges.tex

```diff
@@ -338,6 +345,25 @@
 
   namespace views { inline constexpr @\unspec@ reverse = @\unspec@; }
 
+  // \ref{range.as.const}, as const view
+  template<@\libconcept{input_range}@ R>
+    constexpr auto& @\exposid{possibly-const-range}@(R& r) {          // \expos
+      if constexpr (@\libconcept{constant_range}@<const R> && !@\libconcept{constant_range}@<R>) {
+        return const_cast<const R&>(r);
+      } else {
+        return r;
+      }
+    }
+
+  template<@\libconcept{view}@ V>
+    requires @\libconcept{input_range}@<V>
+  class as_const_view;                                          // freestanding
+
+  template<class T>
+    inline constexpr bool enable_borrowed_range<as_const_view<T>> = enable_borrowed_range<T>;
```

Comment:
Missing `// freestanding` comment. See https://github.com/cplusplus/draft/pull/5713/files#diff-015716b0f124dba6acb7cdf24285de5d954d0958ca8abbb2ad82a015d1852a9eR131, for example.

Reply:
fixed